### PR TITLE
Try consistency of tooltips text

### PR DESCRIPTION
**Pull request by Rahmon**
## Description
Some tooltips have title-cased text, some not. @afercia talked about it in issue #4325. So I've tried make all tooltips text with first character uppercase and the others characters lowercase.  So I changed tooltip component for that.

I also changed the test because with "Help Text" the tooltip text must be "Help text".

## How Has This Been Tested?
This has been tested with "npm test", "npm run test-e2e" and manually on Chrome and Firefox.

## Types of changes
Bug fix

## Checklist:
- [X] My code is tested.
- [X] My code follows the WordPress code style.
- [X] My code follows has proper inline documentation. #

### CODE REVIEW

**Comment by aduth** [comment]
Could we just use [Lodash's `_.upperFirst`](https://lodash.com/docs/4.17.4#upperFirst) instead of implementing our own?

**Comment by Rahmon** [comment]
Yes, sure, we can use this function.

**Review by ahmadawais — APPROVED** [review]

**Review by aduth — APPROVED** [review]
Personally, I'd favor a solution which didn't incur a runtime transform on the text, but rather which could be guarded at a code level via an ESLint rule or equivalent.

That said, this seems like a reasonable interim solution, and I don't expect that there's much of a performance impact 👍

File: components/tooltip/index.js [hunk 1]
```diff
@@ -178,7 +178,7 @@ class Tooltip extends Component {
 					className="components-tooltip"
 					aria-hidden="true"
 				>
-					{ text }
+					{ upperFirst( toLower( text ) ) }
```
Comment (mtias) [inline components/tooltip/index.js]: couldn't this clash with different languages requirements?

Comment (Rahmon) [reply components/tooltip/index.js]: I did some tests with different languages like greek, chinese, korean, russian, persian and hindi. I did not found issues. But it was a quick test. Probably it's necessary investigate more.

Maybe it's better to try via an ESlint rule or equivalent as aduth said.

**Comment by afercia** [comment]
> Personally, I'd favor a solution which didn't incur a runtime transform on the text, but rather which could be guarded at a code level via an ESLint rule or equivalent.

Yep, maybe I'm a bit old-school but I'm not sure I fully understand why this should be solved via code. To me, it's more a language specific issue and maybe good documentation and ESLint are a reasonable approach.

**Pull request merged**